tests/framework/extensions/nodes/utilities.go

```diff
@@ -0,0 +1,10 @@
+package nodes
+
+const (
```

Comment:
Can we call this const.go? I prefer that over the generic utilities name

Reply:
Okay, I've renamed it.